java/src/main/java/com/factset/protobuf/stach/extensions/models/TableData.java

```diff
@@ -21,7 +25,14 @@ public void setRows(List<Row> rows) {
         return metadata;
     }
 
+    public Map<String, List<String>> getMetadataArray() {
+        return metadataArray;
+    }
+
+    public Map<String, List<Value>> getRawMetadata() { return rawMetadata; }
```

Comment:
@saigiridhar21 any suggestions on how do we accommodate both stach v1 & v2. @christine-koul currently is setting v1 metadata in getMetadataArray whereas v2 in getRawMetadata.

Reply:
The issue I'm running into is figuring out how to get the metadata value as type `Value` instead of `MetadataItem`. It easily converts to string but I can't figure out how to convert to `Value`

Reply:
@a-ghag the data types of v1 and v2 MetadataItem are different and using Value type we can have them both accomodated.

@christine-koul in the metadata code, we are getting Value from MetadaItem and stringifying that, may be you can skip the stringifying part if you need to get Value.

Reply:
@Ramanth the MetadaItem doesn't have a method to get Value type. 

Should we just go with Map<String, List<String>> ???

Reply:
How about constructing the Value object with the string value that we are getting in STACH metadata?

Reply:
Can you try this Value.newBuilder().setStringValue(v).build()

Reply:
@a-ghag That works! Thank you